packages/edit-site/src/components/template-details/template-part-area-selector.js

```diff
@@ -28,6 +28,8 @@ export default function TemplatePartAreaSelector( { id } ) {
 
 	return (
 		<SelectControl
+			__nextHasNoMarginBottom
+			className="edit-site-template-details__area-select"
```

Comment:
Are you referring to the 16px of padding only? When trying that out, it looked a bit off: 

![Screen Shot 2022-12-15 at 6 50 39 PM](https://user-images.githubusercontent.com/35543432/208010959-b032cd46-975f-404a-91c6-52d682a5effe.png)

And then, of course there is the issue of not having the update to `TextControl` yet. If we don't keep the margin, should I add the prop to `TextControl` in this case or remove the margin I added here in the future `TextControl` PR? 

![Screen Shot 2022-12-15 at 6 46 35 PM](https://user-images.githubusercontent.com/35543432/208011012-69871dbb-a1c2-4272-9cb1-a5f88cc2444c.png)

Reply:
> Are you referring to the 16px of padding only?

Yes. In cases like this I'm less concerned about whether the bottom whitespace is 16px or 24px, but more about whether the added complexity is warranted. The most straightforward layout is to have 16px whitespace on all sides. If I need to add a new classname and a CSS rule just to make a minor optical adjustment, it better be meaningful and intentional 😄 No strong opinion though.

> And then, of course there is the issue of not having the update to `TextControl` yet. If we don't keep the margin, should I add the prop to `TextControl` in this case or remove the margin I added here in the future `TextControl` PR?

Either is fine with me, but I guess it will look nicer if we also add the prop to TextControl now 👍